Explorer/Assets/DCL/Interaction/Utility/RaycastUtils.cs

```diff
@@ -15,14 +15,14 @@ namespace DCL.Interaction.Utility
     public static class RaycastUtils
     {
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        internal static bool IsSDKLayerInCollisionMask(ColliderLayer layer, ColliderLayer mask) =>
```

Comment:
Since a lot more ASMDEF are being shared now, maybe the internal->public changes are not needed? WDYT?